lib/typescript/httpclient/model/Content.ts

```diff
@@ -8,6 +8,10 @@ export interface Content {
 
 export type RenderedContentUnion = Message | Template | Content;
 
-export function isFromContact(message: RenderedContentUnion) {
-  if (message && 'senderType' in message) return message?.senderType === SenderType.sourceContact;
+export function isFromContact(content: RenderedContentUnion) {
+  if (content && 'fromContact' in content) {
+    return content?.fromContact;
+  } else {
+    return false;
+  }
```

Comment:
can do early return here.

Even though I think this function is odd (as it takes a renderedcontentunion but seems message specific anyway. I think @AitorAlgorta and I will address it in #1466)